### PR TITLE
add number increment input for sync interval

### PR DESCRIPTION
**Pull request by SanjalKatiyar**
https://bugzilla.redhat.com/show_bug.cgi?id=2114501
Screenshot from 2022-10-14 16-49-30

### CODE REVIEW

File: packages/mco/components/disaster-recovery/create-dr-policy/select-replication-type.tsx [hunk 1]
```diff
@@ -38,20 +42,27 @@ const SyncSchedule: React.FC<SyncScheduleProps> = ({ state, dispatch }) => {
...
     const newScheduleFormat = event.target.value;
     setIsOpen(false);
     setSelectedFormat(newScheduleFormat);
     setSyncSchedule(scheduleTime, newScheduleFormat);
   };
 
+  const onChange = (event: React.FormEvent<HTMLInputElement>) => {
+    const target = event.target as HTMLInputElement;
+    const syncTime = isNaN(+target.value) ? minSyncTime : Number(target.value);
+    const normalizedSyncTime = syncTime < minSyncTime ? minSyncTime : syncTime;
```
Comment (GowthamShanmugam) [inline packages/mco/components/disaster-recovery/create-dr-policy/select-replication-type.tsx:62]: The minimum value won't allow Null, also there is no change of syncTime < minSyncTime. Is these two conditions required?

Comment (SanjalKatiyar) [reply packages/mco/components/disaster-recovery/create-dr-policy/select-replication-type.tsx:62]: I can remove 2. if u think it is not needed... then user can only increase value by 1 at a time...

Comment (GowthamShanmugam) [reply packages/mco/components/disaster-recovery/create-dr-policy/select-replication-type.tsx:62]: yes second one is not required, first one is fine

**Review by GowthamShanmugam — APPROVED** [review]

**Pull request merged**